docs/blueprints/EP038.rst

```diff
@@ -0,0 +1,68 @@
+:EP: 38
+:Title: Improving security to coloring application to switches
+:Authors:
+    - Aldo Ortega <aortegay AT fiu DOT edu>
+    - Italo Valcy <idasilva AT fiu DOT edu>
+
+:Created: 2023-06-27
+:Updated: 2023-07-20
+:Kytos-Version: 2023.2
+:Status: Draft
+
+*************************************************************
+EP38 - Improving security to coloring application to switches
+*************************************************************
+
+Abstract
+========
+
+Here it will be explained the vulnerability of **coloring** implementation related ot its broad match field. Also, it is going to be explained how these vulnerabilities could be solved.
+
+Motivation
+==========
+
+The NApp **coloring** installs flows according to a switch neighbors (NNI linked switches). These flows have three crucial characteristics:
+
+  1. The matching criteria has one field, **"dl_src"** which has the same standard as a **MAC** address; ie. **ee:ee:ee:ee:ee:02**.
```

Comment:
Just a heads up that currently [`COLOR_FIELD` is configurable](https://github.com/kytos-ng/coloring/blob/master/main.py#L41), and it's even allowing [`in_port` to be used](https://github.com/kytos-ng/coloring/blob/master/main.py#L138), it might be a good opportunity to reassess which fields should actually be supported from this point forward

Reply:
We need to delete `in_port` since it will be an addition to all NNI-related flows. Also, this configurable option is not really used. We can get away with just using `dl_src`.

Reply:
Only using `dl_src` as a mask. We do not need to support all the other fields.

Reply:
Cool. Right, let's officialize this in the v1 and make sure that in the changelog we also explain that we've restricted and broken compatibility to simplify this NApp and keep it easier to maintain.